tests/solvers/test_sundials.py

```diff
@@ -1085,6 +1103,31 @@ def handle_event(self,solver, event_info):
         sim.simulate(2.5)
         
         nose.tools.assert_almost_equal(sim.t, 2.000000, 4)
+
+    @testattr(stddist = True)
+    def test_terminate_simulation_external_event(self):
+        """
+        This tests the functionality of raising TerminateSimulation exception in handle_result. External event detection.
+        """
+        class Extended_Problem(Implicit_Problem):
+            def __init__(self):
+                pass
```

Comment:
I am curious if the init-function is necessary if the implementation is just `pass`? Don't need to take any action here I only ask because I want to learn :-)

Reply:
This would prevent defaulting to the __init__ function of the parent class.

I went ahead and removed these here anyways, since they are not necessary.

Reply:
Ahh yeah you are right, I did not think of that. Great!